cpp/ql/test/library-tests/templates/nontype_instantiations/general/test.cpp

```diff
@@ -0,0 +1,19 @@
+// semmle-extractor-options: --edg --trap_container=folder --edg --trap-compression=none
+template<int x>
+struct C { };
+
+static const int one1 = 1, one2 = 1;
+C<one1> c = C<one2>();
+C<one1 + one2> e;
```

Comment:
The front-end presents the extractor with the integer constant '2' here with no further information - not even a link back to the source code so we can see how '2' was written.  There is currently no way I can find to tell the extractor about `one1 + one2`.